mindmeld/models/model.py

```diff
@@ -350,12 +365,19 @@ def register_resources(self, **kwargs):  # pylint: disable=no-self-use
     def get_resource(self, name):
         return self._resources.get(name)
 
-    @staticmethod
-    def _get_model_config_save_path(path):
-        head, ext = os.path.splitext(path)
-        model_config_save_path = head + ".config" + ext
-        os.makedirs(os.path.dirname(model_config_save_path), exist_ok=True)
-        return model_config_save_path
+    @property
+    def text_preparation_pipeline(self):
```

Comment:
Unsure why we need this here. Isn't the input to the models already a list of ProcessedQueries?

Reply:
Here's the use-case I found (something that has been there wrt sklearn models classes we have already):
- in `view_extracted_features()`, `predict()`, `predict_proba()`methods of _tagger\_models.py_,  we refer to `self.text_preparation_pipeline` 
- similarly in `get_feature_matrix()` and `inspect()` methods in _text\_models.py_'s sklearn class too.